packages/roosterjs-editor-dom/lib/table/VTable.ts

```diff
@@ -2,14 +2,12 @@ import moveChildNodes from '../utils/moveChildNodes';
 import normalizeRect from '../utils/normalizeRect';
 import safeInstanceOf from '../utils/safeInstanceOf';
 import toArray from '../utils/toArray';
-import { Table, TableFormat, TableOperation, TableSelection, VCell } from 'roosterjs-editor-types';
-
-const TABLE_CELL_SELECTED = '_tableCellSelected';
+import { TableFormat, TableOperation, TableSelection, VCell } from 'roosterjs-editor-types';
 
 /**
  * A virtual table class, represent an HTML table, by expand all merged cells to each separated cells
  */
-export default class VTable implements Table {
+export default class VTable {
     /**
      * The HTML table object
      */

```

Comment:
It seems VTable.selection is not used for now. We can remove it. Later if we still need it, we can add it back

Reply:
Done on latest commit

Reply:
I can see it is still on line 34